### ISSUE TITLE
Error with latest update

### ISSUE THREAD

**Issue by bdew** [hetima/DSP_LSTM#29]
Getting this error constantly with game version 0.10.28.21150 mod version 0.8.5

```
[Warning:  HarmonyX] AccessTools.Field: Could not find field for type UIStatisticsWindow and name entriesLen
System.ArgumentException: FieldRefAccess<UIStatisticsWindow, System.Int32> for Statistics Window (UIStatisticsWindow), entriesLen caused an exception ---> System.MissingFieldException: Field 'UIStatisticsWindow.entriesLen' not found.
  at HarmonyLib.AccessTools.GetInstanceField (System.Type type, System.String fieldName) [0x00010] in <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0 
  at HarmonyLib.AccessTools.FieldRefAccess[T,F] (T instance, System.String fieldName) [0x00032] in <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0 
   --- End of inner exception stack trace ---
  at HarmonyLib.AccessTools.FieldRefAccess[T,F] (T instance, System.String fieldName) [0x00092] in <7f26c0a74c5b43c2a5fc5efd29ec63d6>:0 
  at LSTMMod.UIStatisticsWindowAgent.PostCreate () [0x00023] in <b48996803f5644cbbb92da04353e603c>:0 
  at LSTMMod.LSTM+Patch.UIGame__OnCreate_Postfix () [0x0003f] in <b48996803f5644cbbb92da04353e603c>:0 
  at UIGame._OnCreate () [0x003c9] in <c3b295ce06254256906510939c4119d1>:0 
  at ManualBehaviour._Create () [0x00030] in <c3b295ce06254256906510939c4119d1>:0 
```

```
NullReferenceException: Object reference not set to an instance of an object
  at LSTMMod.LSTM+Patch.Mod_entitySignPool_Postfix (EntitySignRenderer __instance) [0x00007] in <b48996803f5644cbbb92da04353e603c>:0 
  at EntitySignRenderer.Draw () [0x00099] in <c3b295ce06254256906510939c4119d1>:0 
  at FactoryModel.OnCameraPostRender (UnityEngine.Camera cam) [0x000d9] in <c3b295ce06254256906510939c4119d1>:0 
  at (wrapper delegate-invoke) System.Action`1[UnityEngine.Camera].invoke_void_T(UnityEngine.Camera)
  at MainCamera.OnPostRender () [0x00007] in <c3b295ce06254256906510939c4119d1>:0 
```

**Comment by WelshGuyX** [comment]
Getting same error, after a shut down and restarted game after a break today.. No changes to my mods..

NullReferenceException: Object reference not set to an instance of an object
  at LSTMMod.LSTM+Patch.Mod_entitySignPool_Postfix (EntitySignRenderer __instance) [0x00007] in <b48996803f5644cbbb92da04353e603c>:0 
  at EntitySignRenderer.Draw () [0x00099] in <c3b295ce06254256906510939c4119d1>:0 
  at FactoryModel.OnCameraPostRender (UnityEngine.Camera cam) [0x000d9] in <c3b295ce06254256906510939c4119d1>:0 
  at (wrapper delegate-invoke) System.Action`1[UnityEngine.Camera].invoke_void_T(UnityEngine.Camera)
  at MainCamera.OnPostRender () [0x00007] in <c3b295ce06254256906510939c4119d1>:0

**Comment by geeksville** [comment]
yeah - new update probably changed the names of something...

![image](https://github.com/hetima/DSP_LSTM/assets/225513/d5f2fc82-c7be-4f59-b300-cb15c7a93b6d)

**Comment by MortisLegati** [comment]
Confirming

**Comment by geeksville** [comment]
I just made a fix.  Sending it to @hetima in a few minutes.

**Comment by hetima** [comment]
fixed in 0.8.6
thanks
